test/models/Stafftest.js

```diff
@@ -0,0 +1,46 @@
+/**
+ * @file Defines tests for the Student sailsjs model.
+ */
+
+"use strict";
+const helperIntegrationTests = require("./_helpers");
+
+describe("Staff model", function() {
+    let testData = {
+        associations: {},
+        record: {}
+    };
+
+    let testStaff = {
+        username: "TESTSTUDENT@DEWV.EDU",
+        firstName: "TEST",
+        lastName: "STAFF",        
+        isSlpInstructor: "TESTSLP"
+    };
+
+    // Before tests run ...
+    before(async function() {
+
+        // Destroy any old test data that might be haning around
+        await destroyTestData();
+
+        // Create test data: associated model records
+        // Associate, using the new IDs
+
+        // Create main test record, with associations in place
+        testData.record = await Staff.create(testStaff).fetch();
+    });
+
+    // After tests run ...
+    after(destroyTestData);
+
+    // Run helper integration tests
+    helperIntegrationTests("student", testData);
```

Comment:
Another transposition of staff with student.

Reply:
@Emreyildirim1284 It is a common convention with mocha (testing tool) to have the filenames match the filenames of the code being tested. So this file should be `test/models/Staff.js`